src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -1036,6 +1044,7 @@ def __init__(self, streams, stage, areas, focus, ccd, future=None, overlap=0.2,
 
         self.conf_level = 0.8
         self.focusing_method = focusing_method
+        self._use_autofocus: bool = use_autofocus
```

Comment:
Is it useful to explicitly mention it as bool as use_autofocus is defined as bool already?